frontend/src/components/managePermissions/TermsAndConditions.vue

```diff
@@ -0,0 +1,535 @@
+<script lang="ts" setup>
+import AuthService from '@/services/AuthService';
+import { hideTerms, isTermsClosable, isTermsVisible,  } from '@/store/TermsAndConditionsState';
+import Dialog from 'primevue/dialog';
+
+</script>
+<template>
+    <Dialog
+        v-model:visible="isTermsVisible"
+        header="FAM Terms of use"
+        :closable="isTermsClosable"
```

Comment:
so instead of using a global state, pass a property from Profile and ManagePermission